HW_5/task_1.py

```diff
@@ -0,0 +1,26 @@
+"""
+Напишите программу, удаляющую из текста все слова, в которых присутствуют буквы «а», «б» и «в».
+
+Ввод: значение типа <str>
+Вывод: значение типа <str>
+"""
+
+
+# Вариант_1
+def del_word_1(text, sym):
+    return " ".join(filter(lambda x: not (sym[0] in x and sym[1] in x and sym[2] in x), text.split()))
+
+
+# Вариант_2
+def del_word_2(text, sym):
+    return " ".join(filter(lambda x: not all((sym[0] in x, sym[1] in x, sym[2] in x)), text.split()))
```

Comment:
Смотрите, Иван. Ваш код ограничен 3 символами. Но можно немного изменить его и сделать так, что ваш код будет удалять слова с вхождением любых букв.
```suggestion
    return " ".join(filter(lambda x: not all(char in x for char in sym), text.split()))
```